Examples/Tests/multi_J/inputs_rz

```diff
@@ -0,0 +1,135 @@
+# Iterations
+max_step = 50
+
+# Domain
+amr.n_cell = 64 128
+amr.max_level = 0
+warpx.numprocs = 1 2
+
+# Geometry
+geometry.coord_sys = 1
+geometry.prob_lo =   0    -220e-6
+geometry.prob_hi = 200e-6   10e-6
+boundary.field_lo = none damped
+boundary.field_hi = none damped
+
+# Algorithms
+algo.current_deposition = direct
+algo.charge_deposition = standard
+algo.field_gathering = energy-conserving
+algo.maxwell_solver = psatd
+algo.particle_pusher = vay
+algo.particle_shape = 3
+
+# Numerics
+warpx.do_moving_window = 1
+warpx.do_pml = 0
+warpx.moving_window_dir = z
+warpx.moving_window_v = 1.
+warpx.n_rz_azimuthal_modes = 1
+warpx.use_filter = 1
+warpx.verbose = 1
+
+warpx.cfl = 1.
+#warpx.gamma_boost = 1.
+#warpx.boost_direction = z
+
+warpx.do_dive_cleaning = 1
+warpx.do_divb_cleaning = 1
+warpx.do_multi_J = 1
+warpx.do_multi_J_n_depositions = 2
+psatd.J_linear_in_time = 1
+psatd.do_time_averaging = 0
+
+# PSATD
+psatd.update_with_rho = 1
+#psatd.v_galilean = 0. 0. -0.9373391857121336
+
+# Particles
+
+particles.species_names = driver driver_back plasma_e plasma_p
+particles.use_fdtd_nci_corr = 0
+particles.rigid_injected_species = driver
+
+driver.species_type = electron
+driver.injection_style = "gaussian_beam"
+driver.x_rms = 5e-6
+driver.y_rms = 5e-6
+driver.z_rms = 20.1e-6
+driver.x_m = 0.
+driver.y_m = 0.
+driver.z_m = -80e-6
+driver.npart = 1000000
+driver.q_tot = -1e-9
+driver.momentum_distribution_type = "gaussian"
+driver.ux_m = 0.
+driver.uy_m = 0.
+driver.uz_m = 2000.
+driver.ux_th = 4.
+driver.uy_th = 4.
+driver.uz_th = 20.
+driver.zinject_plane = 2e-3
+driver.rigid_advance = true
+driver.projected = true
+driver.focused = false
+driver.initialize_self_fields = 0
+driver.do_symmetrize = 1
+driver.random_theta = 0
```

Comment:
random_theta is not used with the `gaussian_beam` injection style. Is there a reason this was added?